### PR TITLE
Added eslint, prettier, updated expo-cli and extension recommendations for vscode

### PR DESCRIPTION
**Pull request by hubcio2115**
Relates to #6 and my [comment](https://github.com/KuzniaCo/chemistry-app/issues/6#issuecomment-1094106850)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "expo": "~44.0.0",
+    "expo-cli": "^5.3.1",
```
Comment (bartlomiejmont) [inline package.json:14]: expo-cli should be dev-dependency

File: package.json [hunk 2]
```diff
@@ -21,6 +22,13 @@
     "@babel/core": "^7.12.9",
     "@types/react": "~17.0.21",
     "@types/react-native": "~0.64.12",
+    "@typescript-eslint/eslint-plugin": "^5.19.0",
+    "@typescript-eslint/parser": "^5.19.0",
+    "eslint": "^8.13.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.29.4",
+    "prettier": "^2.6.2",
```
Comment (bartlomiejmont) [inline package.json:31]: same here, we don't need It on production so it should be a dev-dep

**Review by bartlomiejmont — CHANGES_REQUESTED** [review]

**Comment by bartlomiejmont** [comment]
Great Idea to add some consistency to our code. Maybe it's a great opportunity to also configure [Husky](https://typicode.github.io/husky/#/)

File: package.json (hunk 1 above)
Comment (hubcio2115) [reply package.json:14]: yes it should, now it will be

File: package.json (hunk 2 above)
Comment (hubcio2115) [reply package.json:31]: but it is

**Comment by hubcio2115** [comment]
> Great Idea to add some consistency to our code. Maybe it's a great opportunity to also configure [Husky](https://typicode.github.io/husky/#/)

What would that add to the project? We already have recommended extensions for vscode that do `prettier --write` automatically. Maybe blocking ability to commit changes with linting errors/warnings?

We would check that anyway with GitHub Actions so I don't see any advantage of that.

**Review by BartoszBrodowski — APPROVED** [review]

**Comment by bartlomiejmont** [comment]
> 



> > Great Idea to add some consistency to our code. Maybe it's a great opportunity to also configure [Husky](https://typicode.github.io/husky/#/)
> 
> What would that add to the project? We already have recommended extensions for vscode that do `prettier --write` automatically. 

>Maybe blocking ability to commit changes with linting errors/warnings?

That's basically what husky does 😄. 

> We would check that anyway with GitHub Actions so I don't see any advantage of that.

It's always better to check that before running the pipeline to avoid wasting resources.

Also, we could maintain good commit message quality.

**Review by bartlomiejmont — APPROVED** [review]
Looks good. Moved Husky implementation to another ticket #34 😉

**Comment by hubcio2115** [comment]
> Looks good. Moved Husky implementation to another ticket #34 😉

Than that means I should read into Husky. Also we could introduce [Volta](https://volta.sh/) so that everyone can be on the same version of Node. But with this we may be comming into too much tooling teritory.

**Comment by bartlomiejmont** [comment]
> 



> > Looks good. Moved Husky implementation to another ticket #34 😉
> 
> Than that means I should read into Husky. Also we could introduce [Volta](https://volta.sh/) so that everyone can be on the same version of Node. But with this we may be comming into too much tooling teritory.

Never heard of that, I am using [NVM ](https://github.com/nvm-sh/nvm#intro) but it looks very similar

**Comment by hubcio2115** [comment]
I've been using NVM until recently, but it wasn't working with me in JetBrains IDEs. So I changed to Volta. IMO it is more intuitive and has things like remembering specific node, npm and yarn versions depending on where you are in a file system.

Also you can specify the versions in package.json so Volta can pick it up and set them for you.

Don't know if NVM has similar features.

**Pull request merged**